src/Rules/LocalOnlyPublicClassMethodRule.php

```diff
@@ -104,19 +104,20 @@ private function isUsedOnlyLocally(
             return true;
         }
 
-        $publicMethodReference = $className . '::' . $methodName;
+        // php method calls are case-insensitive
+        $publicMethodReference = strtolower($className . '::' . $methodName);
 
         if (in_array(
             $publicMethodReference,
-            $localAndExternalMethodCallReferences->getExternalMethodCallReferences(),
+            array_map(fn(string $item) => strtolower($item), $localAndExternalMethodCallReferences->getExternalMethodCallReferences()),
```

Comment:
Could you extract this to a variable? It's pretty complex to read

Reply:
This might be the type missing:

```suggestion
            array_map(fn(string $item): string => strtolower($item), $localAndExternalMethodCallReferences->getExternalMethodCallReferences()),
```